src/monitors/commandHandler.js

```diff
@@ -119,6 +121,7 @@ module.exports = class extends Monitor {
 		this.ignoreOthers = !this.client.user.bot;
 		this.prefixMention = new RegExp(`^<@!?${this.client.user.id}>`);
 		this.prefixMentionLength = this.client.user.id.length + 3;
+		this.regexPrefix = this.client.config.regexPrefix;
```

Comment:
no need to init this, this.client.config.regexPrefix is available when this monitor is constructed.